src/main/java/foodwhere/storage/JsonAdaptedReview.java

```diff
@@ -92,7 +91,7 @@ public Review toModelType(Name modelName) throws IllegalValueException {
 
         final Set<Detail> modelDetails = new HashSet<>(reviewDetails);
 
-        return new Review(new StallName(modelName.fullName), modelDate, modelContent, modelDetails);
+        return new Review(new Name(modelName.fullName), modelDate, modelContent, modelDetails);
```

Comment:
I think we can replace `new Name(modelName.fullName)` with `modelName` since `modelName` is already of type `Name`.

Reply:
> Two minor issues here - open new issue for managing this code debt later in the interest of time. Otherwise looks good to merge.

Thanks for the comment, will open it as a new issue further on.